Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:2
+
+RUN apt-get update && apt-get install -y postgresql-client
+
+COPY requirements.txt /usr/src/app/
+WORKDIR /usr/src/app
+RUN pip install -r requirements.txt
+
+EXPOSE 8000
+
+COPY ampcrowd/docker-entrypoint.sh /usr/src/app/ampcrowd
+ENTRYPOINT ["bash", "ampcrowd/docker-entrypoint.sh"]
+CMD ["-d"]
```

Comment:
Yes/no, its by design really though. A container is kept running as long as its command is running. For example, its common on "ordinary" servers to run `postgres` in the background because there are other services running (like the web server etc). In docker though, a container is supposed to have a single responsibility. Most of the time (as in the case of postgres) that maps to a single, foreground process.

So the change would be to modify `gunicorn_config.py` to run as a foreground process in production mode (or use command line args to override the daemon option).